pkg/terminal/proxy.go

```diff
@@ -181,13 +181,23 @@ func (p *Proxy) HandleProxyEnabled(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	enabled, err := workspaceOperatorIsRunning()
+	isOperatorInstalled, err := operatorIsInstalled()
 	if err != nil {
-		klog.Errorf("Failed to check if workspace operator is running: %s", err)
+		klog.Errorf("Failed to check if the web-terminal operator is installed: %s", err)
 		w.WriteHeader(http.StatusInternalServerError)
 		return
 	}
-	if !enabled {
+	if !isOperatorInstalled {
+		w.WriteHeader(http.StatusServiceUnavailable)
+		return
+	}
+	isOperatorRunning, err := workspaceOperatorIsRunning()
+	if err != nil {
+		klog.Errorf("Failed to check if web-terminal operator is running: %s", err)
```

Comment:
not sure if this shouldn't be:
```suggestion
		klog.Errorf("Failed to check if workspace operator is running: %s", err)
```